srcjs/input_rate.js

```diff
@@ -297,23 +297,27 @@ var InputRateDecorator = function(target) {
   this.inputRatePolicies = {};
 };
 (function() {
-  this.setInput = function(name, value, opts) {
-    this.$ensureInit(name);
+  this.setInput = function(name_type, value, opts) {
+    const {name: inputName} = splitInputNameType(name_type);
+
+    this.$ensureInit(inputName);
 
     if (opts.priority !== "deferred")
-      this.inputRatePolicies[name].immediateCall(name, value, opts);
+      this.inputRatePolicies[inputName].immediateCall(name_type, value, opts);
     else
-      this.inputRatePolicies[name].normalCall(name, value, opts);
+      this.inputRatePolicies[inputName].normalCall(name_type, value, opts);
```

Comment:
@wch should `immediateCall()` and `normalCall()` be called with `name_type` or `inputName`?

Reply:
It invokes `$doSetInput`, so it should be called with `inputName`.